huaweicloud/services/acceptance/acceptance.go

```diff
@@ -196,83 +196,137 @@ func TestCheckResourceAttrWithVariable(resourceName, key, varStr string) resourc
 		// Get the value based on resName and keyName from the state.
 		rs, ok := s.RootModule().Resources[resName]
 		if !ok {
-			return fmtp.Errorf("Can't find %s in state : %s.", resName, ok)
+			return fmtp.Errorf("Can't find %s in state : %v.", resName, ok)
 		}
 		value := rs.Primary.Attributes[keyName]
 
 		return resource.TestCheckResourceAttr(resourceName, key, value)(s)
 	}
 }
 
-// CheckResourceDestroy check whether resources destroied in HuaweiCloud.
+func (rc *resourceCheck) checkResourceDestroy(s *terraform.State) error {
+	strs := strings.Split(rc.resourceName, ".")
+	var resourceType string
+	for _, str := range strs {
+		if strings.Contains(str, "huaweicloud_") {
+			resourceType = strings.Trim(str, " ")
+			break
+		}
+	}
+
+	if resourceType == "" {
+		return fmtp.Errorf("The format of the HuaweiCloud resource name is invalid, please check your configuration.")
+	}
+
+	conf := TestAccProvider.Meta().(*config.Config)
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != resourceType {
```

Comment:
`CheckResourceDestroy` does not need to be changed, it checks all resources that matched the `resourceType`

Reply:
yeah, this function has been restored.